coriolis/api/common.py

```diff
@@ -10,3 +10,8 @@ def get_paging_params(req):
     if limit is not None:
         limit = utils.parse_int_value(limit)
     return marker, limit
+
+
+def get_force_param(req):
+    force_param = req.GET.get('force', '')
+    return (force_param.lower() == "true")
```

Comment:
How sure are we that the `webob` lib we use will always returns strings?
Please add a `str` cast for safety.